src/constants.js

```diff
@@ -166,3 +166,169 @@ export const MAIN_MARKETPLACE_CONTRACT = MARKETPLACE_CONTRACT_TEIA // the one th
 export const MAIN_MARKETPLACE_CONTRACT_SWAP_TYPE = SWAP_TYPE_TEIA
 
 export const BURN_ADDRESS = 'tz1burnburnburnburnburnburnburjAYjjX'
+
+export const COVER_COMPRESSOR_OPTIONS = {
+  quality: 0.85,
+  maxWidth: 1024,
+  maxHeight: 1024,
+}
+
+export const THUMBNAIL_COMPRESSOR_OPTIONS = {
+  quality: 0.85,
+  maxWidth: 350,
+  maxHeight: 350,
+}
+
+export const LICENSE_TYPES = {
+  none: 'None (All rights reserved)',
+  'cc-by-4.0': 'CC-BY-4.0 (Attribution)',
+  'cc-by-sa-4.0': 'CC BY-SA 4.0 (Attribution ShareAlike)',
+  'cc-by-nd-4.0': 'CC BY-ND 4.0 (Attribution-NoDerivs)',
+  'cc-by-nc-4.0': 'CC BY-NC 4.0 (Attribution-NonCommercial)',
+  'cc-by-nc-sa-4.0': 'CC BY-NC-SA 4.0 (Attribution-NonCommercial-ShareAlike)',
+  'cc-by-nc-nd-4.0': 'CC BY-NC-ND 4.0 (Attribution-NonCommercial-NoDerivs)',
+  custom: 'Custom (Specify below)',
+}
+
+export const LICENSE_TYPES_OPTIONS = Object.keys(LICENSE_TYPES).map((k) => {
```

Comment:
afaik javascript does not guarantee a specific order of the object keys, but in practice, browsers order them as you would expect ('none' first).

Reply:
I did not know that 🙈 ! I guess we could ensure it with and `_.orderBy` but I tested a few browsers and it seemed consistent indeed (browser odering them)

Reply:
yes, no need to change it, in practice it's not a problem. a Map guarantees the order btw, but that is clunky to use :)